packages/ihe-gateway/server/CodeTemplates/Common Library/Config/Config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This module provides a standardized way to access environment variables.
+ */
+
+function getEnvVar(name) {
+  const value = java.lang.String(Packages.java.lang.System.getenv(name));
+  if (value == null) {
+    throw new Error(`Environment variable ${name} is not defined.`);
+  }
+  return value;
+}
+
+const Config = {
+  getSamlHomeCommunityId: function () {
+    return getEnvVar("SAML_HOME_COMMUNITY_ID");
+  },
+
+  getInboundXcpdUrl: function () {
+    return getEnvVar("INBOUND_PATIENT_DISCOVERY_URL");
+  },
+
+  getInboundDqUrl: function () {
+    return getEnvVar("INBOUND_DOCUMENT_QUERY_URL");
+  },
+
+  getInboundDrUrl: function () {
```

Comment:
let's perhaps be more explicit with these function names so we have an easier time searching the codebase:
```
getInboundPatientDiscoveryURL()
getInboundDocumentQueryURL()
getInboundDocumentRetrievalURL()
```

will also address the comment https://github.com/metriport/metriport/pull/1521/files#r1482351731